test/integration/url-imports/test/index.test.js

```diff
@@ -102,13 +102,44 @@ describe(`Handle url imports`, () => {
               browser
                 .elementByCss('#static-css')
                 .getComputedCss('background-image'),
-            /^url\("http:\/\/localhost:\d+\/_next\/static\/media\/vercel\.[0-9a-f]{8}\.png"\)$/
+            `url("https://github.com/vercel/next.js/raw/canary/test/integration/url/public/vercel.png")`
```

Comment:
That's not what should happen here. By using `urlImports` is should bundle the `url()` referenced resource. So it must have an url pointing to localhost. If this results in an external url the test is actually broken.